### ISSUE TITLE
Missing (incorrect) card art for several MSG cards

### ISSUE THREAD

**Issue by AshanJiang** [HearthSim/Hearthstone-Deck-Tracker#2982]
![qq 20161130235351](https://cloud.githubusercontent.com/assets/17903899/20759590/44d7a852-b758-11e6-9b10-395ce124dc5c.png)
As shown in the screenshot the Jade Blossom image is not correct.
v1.1.2

**Comment by jleclanche** [comment]
@azeier that's the 404 for the image. did you grab it too early or is it just missing? cc @andburn

**Comment by azeier** [comment]
Jup, https://art.hearthstonejson.com/v1/tiles/CFM_713.png seems to be missing.

**Comment by jleclanche** [comment]
@andburn the algorithm in [generate_card_textures.py](https://github.com/HearthSim/HearthstoneJSON/blob/master/generate_card_textures.py#L43) is not picking up on the tile. It's generating the card just fine, though.

This is what the CardDef looks like:

```json
{'m_SocketInEffectOpponentPhone': '', 'm_SocketInOverrideHeroAnimation': 0, 'm_GoldenCustomDeathSpellPath': '', 'm_HistoryTileHalfPortrait': ObjectPointer(file_id=0, path_id=-7053269751578355351), 'm_Enabled': 1, 'm_AttackEffectDef': {'m_SoundSpellPaths': [], 'm_SpellPath': ''}, 'm_CustomHeroArmorSpell': '', 'm_PracticeAIPortrait': None, 'm_EnchantmentPortrait': None, 'm_AnnouncerLinePath': '', 'm_DeckPickerPortrait': {'screens': [4], 'values': [None]}, 'm_DeathEffectDef': {'m_SoundSpellPaths': [], 'm_SpellPath': ''}, 'm_CustomDeckPortrait': {'screens': [4], 'values': [None]}, 'm_SocketInEffectFriendlyPhone': '', 'm_GoldenCustomSpawnSpellPath': '', 'm_HistoryTileFullPortrait': ObjectPointer(file_id=0, path_id=1602512738394650720), 'm_Script': ObjectPointer(file_id=0, path_id=8946025770441738808), 'm_CustomHeroTraySettings': [], 'm_Name': '', 'm_LifetimeEffectDef': {'m_SoundSpellPaths': [], 'm_SpellPath': ''}, 'm_SuppressDeathrattleDeath': 0, 'm_AnnouncerLineAfterVersusPath': '', 'm_PlayEffectDef': {'m_SoundSpellPaths': [], 'm_SpellPath': 'Assets/Shared/Effects/Spells/Nature/AreaEffect/Nature_Untargeted_WildGrowth'}, 'm_DeckBoxPortrait': None, 'm_CustomSummonSpellPath': 'Assets/Shared/Effects/Spells/Custom_Summon/JadeGolem_CustomSummon', 'm_TriggerEffectDefs': [], 'm_CustomHeroPhoneTray': '', 'm_AlwaysRenderPremiumPortrait': 0, 'm_SocketInParentEffectToHero': 1, 'm_CustomDeathSpellPath': '', 'm_SocketInEffectFriendly': '', 'm_SuppressPlaySoundsOnSummon': 0, 'm_CustomHeroPhoneManaGem': '', 'm_DeckCardBarPortrait': None, 'm_GoldenCustomSummonSpellPath': '', 'm_CustomSpawnSpellPath': 'Assets/Shared/Effects/Spells/Custom_Summon/JadeGolem_CustomSummon', 'm_CollectionHeroDefPath': '', 'm_CardTextBuilderType': 1, 'm_CustomHeroTray': '', 'm_CustomKeywordSpellPath': '', 'm_SubOptionEffectDefs': [], 'm_GameObject': ObjectPointer(file_id=0, path_id=8797026263226253757), 'm_SocketInEffectOpponent': '', 'm_PremiumPortraitTexturePath': '', 'm_PortraitTexturePath': 'Assets/Game/Cards/017 Gangs/CFM_713/HS9-187_S.psd', 'm_PremiumPortraitMaterialPath': 'Assets/Game/Cards/017 Gangs/CFM_713/CFM_713_PremiumCard.mat', 'm_EmoteDefs': [], 'm_AnnouncerLineBeforeVersusPath': ''}
```

**Comment by andburn** [comment]
looking into it now

**Comment by jleclanche** [comment]
I pushed a change which skips texture lookups for cards that are not in `--only`, so you can easily debug this with `./generate_card_textures.py --only=CFM_713 ~/.local/games/Hearthstone/Data/Win/card*`.

**Comment by azeier** [comment]
This should be fixed with the latest update (1.1.2.1).
